app/forms/hyrax/date_terms_behavior.rb

```diff
@@ -0,0 +1,11 @@
+module Hyrax
```

Comment:
I think this is something we should namespace to `ScholarsArchive` instead of `Hyrax`